docs/source/filter.mdx

```diff
@@ -0,0 +1,33 @@
+# Filter rows in a dataset
+
+Datasets Server provides a `/filter` endpoint for filtering rows in a dataset.
+
+<Tip warning={true}>
+  Currently, only <a href="./parquet">datasets with Parquet exports</a>
+  are supported so Datasets Server can index the contents and run the search without
+  downloading the whole dataset.
+</Tip>
+
+This guide shows you how to use Datasets Server's `/filter` endpoint to filter rows based on a query string.
+Feel free to also try it out with [ReDoc](https://redocly.github.io/redoc/?url=https://datasets-server.huggingface.co/openapi.json#operation/filterRows).
+
+The `/filter` endpoint accepts five query parameters:
```

Comment:
```suggestion
The `/filter` endpoint accepts six query parameters:
```

or remove the value (more robust to future edits)

Reply:
Done.